packages/v/view-ui-plus.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "view-ui-plus",
+  "description": "An enterprise-level UI component library and front-end solution based on Vue.js 3",
+  "homepage": "www.iviewui.com",
+  "keywords": [
+    "iview",
+    "vue",
+    "viewui",
+    "viewuiplus",
+    "vue.js",
+    "component",
+    "components",
+    "ui",
+    "framework"
+  ],
+  "authors": [
+    {
+      "name": "Aresn",
+      "email": "admin@aresn.com",
+      "url": "https://www.iviewui.com"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/view-design/ViewUIPlus.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "view-ui-plus",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
Please avoid using (open) globstars, as it can lead to files being included on the CDN that aren't needed, or that we don't support.

Looking at the package, I suspect you want `{,locale/}*.js?(.map)`, `styles/*.css`, and `styles/fonts/*.@(svg|ttf|woff|woff2)`?

Reply:
Updated, thx u.